plugins/htpasswd.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: htpasswd
+spec:
+  version: v0.1.6
+  homepage: https://github.com/shibumi/kubectl-htpasswd
+  shortDescription: Create nginx-ingress compatible basic-auth secrets
+  description: |
+    This plugin provides a convenient way to create nginx-ingress compatible
+    basic auth secrets.
+  caveats: ""
```

Comment:
nit:You can drop the field if unused.

Reply:
let me know if I should submit as is, or whether you want to fix it :)

Reply:
Hi @ahmetb  Sorry, I have been rather busy with my master thesis the last week. I will remove caveats line asap and will get back to this PR :)